duckreg/estimators.py

```diff
@@ -108,13 +160,15 @@ def bootstrap(self):
             )
             df_boot.eval(create_means, inplace=True)
 
-            y = df_boot[f"mean_{self.outcome_vars[0]}"].values
-            X = df_boot[self.strata_cols].values
-            X = np.c_[np.ones(X.shape[0]), X]
-            n = df_boot["count"].values
-            boot_coefs[b, :] = wls(X, y, n)
+            y, X, n = self.collect_data(data=df_boot)
```

Comment:
this is an reasonable choice [use the leanest possible solver in the bootstrap loop regardless of solver choice in point estimation]